plugins/http_plugin/tests/unit_tests.cpp

```diff
@@ -622,6 +622,12 @@ BOOST_FIXTURE_TEST_CASE(bytes_in_flight, http_plugin_test_fixture) {
       return count_of_status_replies;
    };
 
+   auto wait_for_no_bytes_in_flight = [&](uint16_t max = std::numeric_limits<uint16_t>::max()) {
+      while (http_plugin->bytes_in_flight() > 0 && http_plugin->requests_in_flight() > 0 && --max)
```

Comment:
When waiting for no bytes in flight why do we need to look at requests in flight too?

Reply:
No "need", just thought it was good to also verify `requests_in_flight` appropriately hits 0.